src/Altinn.App.Core/Internal/Patch/IPatchService.cs

```diff
@@ -1,45 +0,0 @@
-using Altinn.App.Core.Features;
-using Altinn.App.Core.Models;
-using Altinn.App.Core.Models.Result;
-using Altinn.App.Core.Models.Validation;
-using Altinn.Platform.Storage.Interface.Models;
-using Json.Patch;
-
-namespace Altinn.App.Core.Internal.Patch;
-
-/// <summary>
-/// Service for handling JsonPatches to data elements.
-/// </summary>
-public interface IPatchService
```

Comment:
This along with `InternalPatchService` doesn't feel great. Have you run a search to make sure noone is referencing this?

Reply:
It's been pretty heavily changed in 8.5.0-x anyway, so using it won't be problem free.